src/aleph_client/asynchronous.py

```diff
@@ -64,9 +65,9 @@ def get_fallback_session() -> ClientSession:
 
 
 async def ipfs_push(
-        content,
-        session: Optional[ClientSession] = None,
-        api_server: str = settings.API_HOST,
```

Comment:
The thing about formatting:
For some reason, PyCharm believes the double indent to be the superior one, probably based on [PEP 8 - Indentation](https://peps.python.org/pep-0008/#indentation)
I found it weird at first, too, but after some consideration, I actually prefer this style. The list of parameters is perfectly identifiable against the regular function body, which I find satisfying to look at.

Reply:
So, as discussed, let's use black for all our Python codebases to avoid formatting debates.